ember/app/templates/map/moderations/index.hbs

```diff
@@ -7,6 +7,32 @@
     {{/link-to}}
   </div>
 
+  <div class="panel-headers">
+    <h1>Flags</h1>
+    <h3>Listed below are all user-created flags for you to review</h3>
+  </div>
+
+  {{#if filteredFlags}}
+    <ul class="item-list">
+      <li class="list-header">
+        <div class="row">
+          <div>Development</div>
+        </div>
+      </li>
+      <div class='list-body'>
+        {{#each filteredFlags as |flagged-development|}}
+          <li data-mod-id={{flagged-development.id}}>
+            <div class="row">
+              <div>
+                {{#link-to 'map.developments.development' flagged-development}}{{flagged-development.name}}{{/link-to}}
+              </div>
+            </div>
```

Comment:
We should have a button here that allows the admin to "unflag" the development from this panel.